_docs/management/leveraging-authentication.md

```diff
@@ -92,13 +92,14 @@ access token. Here is where things get fun.
 
 3.  The `access_token` is a JSON Web Token that can be decoded using a
     library such as [node-jsonwebtoken](https://github.com/auth0/node-jsonwebtoken).
-    See https://jwt.io/ for a list of libraries for various languages. Decode it
-    to get the authenticated user's `email`, which you can then use within
+    See https://jwt.io/ for a list of libraries for various languages. 
+    
+    Verify the token's signature using cloud.gov's JWK Set and the RSA256 alogrithm. This step ensures the
+    token is authentic. The JWK Set for cloud.gov's UAA is located at `https://uaa.fr.cloud.gov/token_keys`. 
```

Comment:
I wanted to make this seem less like an optional step, and also make the language more google-able for those of use who are new to implementing authentication!